bots/example_bots/helpers/grid.py

```diff
@@ -27,90 +27,156 @@ class Grid:
 
     def __init__(
         self,
+        # Assets
+        pair_name: str,
         # Inventory
-        starting_base_asset_amount: Decimal,
-        starting_quote_asset_amount: Decimal,
-        starting_base_asset_average_price: Optional[Decimal],
+        starting_base_asset_amount: Decimal | str,
```

Comment:
we should add a check here to ensure that the provided wallet has enough funds to actually cover this starting amount. currently, if the user inputs a value that is > than their actual wallet holding it will cause the bot to throw errors: 

`ERROR:root:execution reverted: ERC20: transfer amount exceeds balance`

Reply:
@denverbaumgartner Have added this in: https://github.com/RubiconDeFi/rubi-py/pull/76